tests/dialects/test_oracle.py

```diff
@@ -313,3 +313,11 @@ def test_connect_by(self):
 
         for query in (f"{body}{start}{connect}", f"{body}{connect}{start}"):
             self.validate_identity(query, pretty, pretty=True)
+
+    def test_variance(self):
+        self.validate_all("VAR_SAMP(x)", write={"oracle": "VAR_SAMP(x)"})
+        self.validate_all("VAR_POP(x)", write={"oracle": "VAR_POP(x)"})
+        self.validate_all("VARIANCE(x)", write={"oracle": "VAR_SAMP(x)"})
+        self.validate_all(
+            "VAR_POP(x)", read={"": "VARIANCE_POP(x)"}, write={"oracle": "VAR_POP(x)"}
+        )
```

Comment:
Let's use `self.validate_identity` for the first three tests since we're only doing a roundtrip, or even better keep the `validate_all` and augment the test to check how this is transpiled from and to Oracle (e.g. check Snowflake, DuckDB, ...).

Also, a nit / small note: I know that this isn't very consistent in the codebase, but I try to format these `validate_all` tests as:

```python
self.validate_all(
    ...,
    read={
        ...,
    },
    write={
        ...,
    },
)
```

This reads a bit better imo.